app/services/idv/actions/verify_document_action.rb

```diff
@@ -44,8 +44,13 @@ def enqueue_job
           'encryption_key', 'front_image_iv', 'back_image_iv', 'selfie_image_iv', 'front_image_url',
           'back_image_url', 'selfie_image_url'
         ).to_h
+        applicant_pii = flow_session[:pii_from_doc]
+        applicant = {
+          applicant_pii: applicant_pii,
```

Comment:
part of me thinks it would be nice to only do the UUID/prefix here instead of all the applicant PII because it gets persisted to the queue (even if temporarily) it would be nice to handle the minimum PII possible in the job

Reply:
I'd agree. Zach's reason is the the most relevant, but I think we should also handle default parameter values  outside of the job if possible.

Mostly an informational note, but this change isn't backwards compatible and will cause jobs to fail in the transition. I don't think we need to do the process of supporting both right now though, it's fine since we're limited to sandboxes.

Reply:
> Mostly an informational note, but this change isn't backwards compatible and will cause jobs to fail in the transition. I don't think we need to do the process of supporting both right now though, it's fine since we're limited to sandboxes.

I had the same concern, so I took a look atthis, and I think that the job arguments/encrypted payload do keep the same structure, that `decrypted_args[:applicant_pii]` may be nil, which is handled, so I think this is safe across deploys

Reply:
ah right, thanks. I should have known since I think I implemented that part of it.